.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -0,0 +1,119 @@
+name: 🐞 bug fix
+description: "file a bug in noop services"
+title: "[Bug]: "
+labels: ["bug"]
+assignees:
+  - noop-cloud/noop
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## Thanks for taking the time to fill out this bug report!
+  - type: markdown
+    attributes:
+      value: |
+        - Please review open issues labeled as `bugs` to ensure your issue is not a duplicate.
+        - For general questions, ideas and concerns that are not related to a bug, visit our [discussions page](https://github.com/noop-cloud/supreme-garbanzo/discussions)
+  - type: input
+    id: contact
+    attributes:
+      label: Contact Details
+      description: How can we get in touch with you if we need more info?
+      placeholder: ex. email@example.com
+    validations:
+      required: false
+  - type: dropdown
+    id: affected-os
+    attributes:
+      label: Operating System
+      description: What Operating System(s) have you experienced this bug on?
+      multiple: true
+      options:
+        - Windows
+        - MacOS
+    validations:
+      required: false
+  - type: dropdown
+    id: affected-arch
+    attributes:
+      label: Architecture
+      description: What machine architecture(s) have you experienced this bug on?
+      multiple: true
+      options:
+        - amd64
+        - arm64
+    validations:
+      required: false
+  - type: dropdown
+    id: affected-browsers
+    attributes:
+      label: Affected Browsers
+      description: If using the web application, which browsers have you experiened this bug in?
+      multiple: true
+      options:
+        - Firefox
+        - Chrome
+        - Safari
+        - Microsoft Edge
```

Comment:
Someone could be using a browser other than the ones listed here.

Reply:
i can switch this, keep in mind, the idea behind lists was to control inputs from the public. We can open it up and let people type w/e too.

Reply:
I'm open to any ideas here on how to implement a more flexible list. I just don't think hardwiring a static list of browser options is practical, as this setup does not provide an alternative input if a user is using a browser that is not listed.

Reply:
switched to the text area for now. It may just be something we have to deal with if/when it happens